3_javascript/javascript1/week3/Homework/script.js

```diff
@@ -0,0 +1,219 @@
+// warm up work item array removal //
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+
+console.log(names.at(1));
+
+// trying other parameters //
+
+console.log(names.indexOf("Rasmus"));
+
+console.log(names[names.length - 1]);
+
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//  When will we be there //
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+const averageSpeedToArrive = function (num) {
+  let hours = Math.floor(num / 60);
+  if (hours.toString().length === 1) {
+    hours = "0" + hours;
+  }
+
+  let minutes = num % 60;
+  if (minutes.toString().length === 1) {
+    minutes = "0" + minutes;
+  }
+
+  return `${hours}:${minutes}`;
+};
+
+const travelTime = averageSpeedToArrive(travelInformation);
+console.log(averageSpeedToArrive(432)); // 8 hours and 38 minutes
+
+// Series duration of my life //
+
+const seriesDurations = [
+  {
+    title: "Friends",
+    days: 3,
+    hours: 23,
+    minutes: 0,
+  },
+  {
+    title: "Hannah Montana",
+    days: 5,
+    hours: 12,
+    minutes: 57,
+  },
+  {
+    title: "Harry Potter",
+    days: 5,
+    hours: 12,
+    minutes: 57,
+  },
+];
+
+const seriesDurationsOfMyLife = function (seriesDuration) {
```

Comment:
her you don't need series parameter because you have it as constant so you can use it directly in your function

Reply:
noted